crates/ruff_python_formatter/tests/snapshots/format@expression__number.py.snap

```diff
@@ -25,4 +25,17 @@ input_file: crates/ruff_python_formatter/resources/test/fixtures/ruff/expression
 ```
 
 
+## Preview changes
+```diff
+--- Stable
++++ Preview
+@@ -1,4 +1,5 @@
+ 0.1
++
```

Comment:
This should also be fixed by #8216

Reply:
Trying to understand, why / how is this happening?

Reply:
Ah, I see, there's a bug in `try_from_docstring` that's fixed in that PR. I'm just gonna copy that fix in here -- seems odd to commit without it.

Reply:
my plan was to merge the other one first and then rebase, but this does too